bootstrap-salt.sh

```diff
@@ -3308,16 +3292,15 @@ install_fedora_stable_post() {
 }
 
 install_fedora_git_deps() {
-    __fedora_get_package_manager
 
     if [ "$_INSECURE_DL" -eq $BS_FALSE ] && [ "${_SALT_REPO_URL%%://*}" = "https" ]; then
-        $FEDORA_PACKAGE_MANAGER ca-certificates || return 1
+        dnf ca-certificates || return 1
```

Comment:
I think the above line should be
```bash
dnf install -y ca-certificates || return 1
```
Right?

It seems that the `install` parameter was accidentally removed in the previous version.